frontend/packages/console-plugin-sdk/src/hooks.ts

```diff
@@ -0,0 +1,52 @@
+import * as _ from 'lodash';
+import { useMemo } from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector, shallowEqual } from 'react-redux';
```

Comment:
FYI, we're using an outdated version of `@types/react-redux`

```bash
$ yarn why @types/react-redux
=> Found "@types/react-redux@6.0.2"

$ yarn why react-redux
=> Found "react-redux@7.1.0"
```

I've tried to bump it to match `react-redux` but ran into typing errors, so using `@ts-ignore` comment for now.